packages/core/components/DropZone/index.tsx

```diff
@@ -237,6 +238,15 @@ function DropZoneEdit({ zone, allow, disallow, style }: DropZoneProps) {
                 const label =
                   componentConfig?.["label"] ?? item.type.toString();
 
+                const canDrag = getPermissions({
+                  selectedItem: getItem(
+                    { index: i, zone: zoneCompound },
+                    appContext.state.data
+                  ),
+                  config: appContext.config,
+                  globalPermissions: appContext.globalPermissions || {},
+                }).drag;
```

Comment:
I think we should remove the `drag` permissions based on our conversation yesterday

If you disable drag on an item, you don't disable drag on the other items, so it serves no purpose.